### DISCUSSION TITLE
Default pages

### DISCUSSION THREAD

**Discussion by ugintl** [rappasoft/laravel-boilerplate#1563] [Q&A]
I want to do 2 things with this boilerplate. 1. When I open the site, it should go to login page. 2. When I login, it should take me to dashboard. How can I do it?

**Comment by rappasoft** [comment]
1. Override this route to just go to the login page if the user isn't logged in: https://github.com/rappasoft/laravel-boilerplate/blob/master/routes/frontend/home.php#L11
2. It takes you to the dashboard already when you login based on your role: https://github.com/rappasoft/laravel-boilerplate/blob/master/app/Helpers/Global/GeneralHelper.php#L38

There is nothing special about this project, it is basic Laravel. Just follow the routes to the controllers and you should be able to do what you want to do.
